### PR TITLE
Add documentation for the explosion damage function

### PR DESCRIPTION
**Pull request by link-discord**
see title

### CODE REVIEW

**Review by rom1504 — CHANGES_REQUESTED** [review]
Can you also add the new event

**Comment by link-discord** [comment]
what new event is it?

**Review by u9g — CHANGES_REQUESTED** [review]
can you add the new event to the typings too?

**Comment by link-discord** [comment]
shake my head

**Comment by link-discord** [comment]
> can you add the new event to the typings too?

Done

**Comment by u9g** [comment]
lol you forgot the `bot.getExplosionDamages` in typings

File: docs/api.md [hunk 1]
```diff
@@ -1090,6 +1092,7 @@ Fires when your hp or food change.
 
 Fires when your oxygen level change.
 
+#### "entityAttributes" (entity)
```
Comment (rom1504) [inline docs/api.md:1095]: Can you add a description ?

Comment (link-discord) [reply docs/api.md:1095]: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Review by rom1504 — APPROVED** [review]

**Pull request merged**